source/_patterns/01-elements/link/link.scss

```diff
@@ -31,46 +37,43 @@
 		}
 	}
 
-	// Sizes
-	&%size-Small {
-		&:not(.is-icon-text-replace) {
-			font-size: to-rem($pxValue: 14);
-		}
-
-		&[rel],
-		&[data-icon],
-		&[data-icon-before] {
-			@include icon-meta(20);
-		}
-
-		&[data-icon-after]::after {
-			@include icon-meta($size: 20, $position: "after");
-		}
+	&[data-content] {
+		--icon-font-family: var(--db-base-icon-font-family);
+		--icon-font-size: var(--db-base-icon-font-size);
 	}
 
-	&[data-size="small"] {
-		@extend %size-Small;
+	&[data-content="internal"] {
+		@include icon(glyph(link), 24, "outline", "after");
 	}
 
-	// Define the available icons
-	&[rel="configuration"] {
-		@include icon(glyph(settings), 24, "outline", $partial: $partial);
+	&[data-content="external"] {
+		@include icon(glyph(link-external), 24, "outline", "after");
 	}
 
-	&[rel="messages"] {
-		@include icon(glyph(chat), 24, "outline", $partial: $partial);
+	&[data-content="internal"],
+	&[data-content="external"] {
+		&::after {
+			--icon-margin-before: var(--db-spacing-fixed-2xs);
+			margin-top: -3px;
+		}
+
+		&:focus-visible {
+			outline-offset: -1px;
+			padding: 0 2px;
+		}
```

Comment:
This leads to that the links is actually increasing in width on focus (compare to attached video) – is this intended ?

https://user-images.githubusercontent.com/787658/211268848-d9097cca-1d8f-43af-96ce-ce182e2018aa.mov

Reply:
fixed in 6912f57271c0b6ed3e49daa5d1499b48e08dda69